mcs/packages/Makefile

```diff
@@ -12,15 +12,63 @@ ROSLYN_FILES = \
 	$(ROSLYN_CSC_DIR)/System.Collections.Immutable.dll	\
 	$(ROSLYN_CSC_DIR)/System.Reflection.Metadata.dll
 
-DISTFILES = $(ROSLYN_FILES)
+ROSLYN_FILES_FOR_MSBUILD = \
+	$(ROSLYN_CSC_DIR)/Microsoft.Build.Tasks.CodeAnalysis.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CSharp.Core.targets \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.CSharp.Scripting.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.CSharp.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.Scripting.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.VisualBasic.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.CodeAnalysis.dll \
+	$(ROSLYN_CSC_DIR)/Microsoft.VisualBasic.Core.targets \
+	$(ROSLYN_CSC_DIR)/Microsoft.Win32.Primitives.dll \
+	$(ROSLYN_CSC_DIR)/System.AppContext.dll \
```

Comment:
With the facades removed from the `Roslyn/` folder, I can’t run `csi.exe` because of missing `System.ValueTuple` and `System.Runtime`. If I local copy those from `lib/mono/4.5`, then it works.